common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/FetchShuffleBlocks.java

```diff
@@ -62,11 +62,9 @@ public FetchShuffleBlocks(
 
   @Override
   public String toString() {
-    return toStringHelper()
-      .append("mapIds", Arrays.toString(mapIds))
-      .append("reduceIds", Arrays.deepToString(reduceIds))
-      .append("batchFetchEnabled", batchFetchEnabled)
-      .toString();
+    return "FetchShuffleBlocks[appId=" + appId + ",execId=" + execId + ",shuffleId=" + shuffleId +
+        ",mapIds=" + Arrays.toString(mapIds) + ",reduceIds=" + Arrays.deepToString(reduceIds) +
+        ",batchFetchEnabled=" + batchFetchEnabled + "]";
```

Comment:
This adds more fields than before?

Reply:
It came from `toStringHelper()` method at line 65. Technically, this PR removed `toStringHelper` usage.

https://github.com/apache/spark/blob/c7176246a21aee6b48dfe957f2baf4e547a48e4b/common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/AbstractFetchShuffleBlocks.java#L46-L51